### ISSUE TITLE
Redundancy in MVC topic sections

### ISSUE THREAD

**Issue by robbert229** [aspnet/Docs#591]
Right now with the MVC documentation there is section labeled **getting started** and one named **tutorials** what is the major difference between them? From my point of view both sections cover the same content just have different names.

**Comment by robbert229** [comment]
Example inside of Tutorials there is an article named **Building Your First MVC 6 Application**. The title at the top of the page is **Get Started with EF7 using MVC6**. Then in the getting started section there is only stubs for **Building Your First MVC 6 Application**

**Comment by robbert229** [comment]
@MikeWasson @Rick-Anderson ^

**Comment by danroth27** [comment]
Yeah, we should be consistent with what we do in the ASP.NET project. Getting Started should cover installation and Tutorials should be, well, tutorials :smile:.

**Comment by danroth27** [comment]
Note that the MVC installation instructions should assume ASP.NET 5 was already installed. So, it's basically just installing the right package.

Something like what expressjs has would make sense: http://expressjs.com/starter/installing.html

**Comment by robbert229** [comment]
That's what I was thinking. I was confused while looking through. Do you want me to merge it (similar to expressjs) and make a pr?

**Comment by robbert229** [comment]
@danroth27 Right now I am in the process of merging the two sections. My question is do we need both **Get Started with Entity Framework 7 using ASP.NET MVC 6** and **Building Your First MVC 6 Application**? The latter tutorial is only a stub, and to my understanding is the same to the first tutorial.

**Comment by Rick-Anderson** [comment]
Hey @robbert229  I'm almost finished with Building Your First MVC 6 Application. Can you send me an email and I'll send you a copy of it - or I can do a PR and you can build it. I'd like you feedback.

**Comment by Rick-Anderson** [comment]
@robbert229  There is considerable overlap between the two tutorials and we'll think about fixing that for RTM, but for now I think we'll keep both of them.

**Comment by robbert229** [comment]
What is your email?

**Comment by Rick-Anderson** [comment]
rick.anderson@microsoft.com

**Comment by robbert229** [comment]
Email sent.

**Comment by robbert229** [comment]
Also in the near future I will probably need some feedback on the razor guide which I am currently working on, so I will appreciate it if I can get your opinion then.

**Comment by Rick-Anderson** [comment]
@robbert229  NP, let me know

**Comment by danroth27** [comment]
We've decided to leave the "Your First  . . ." articles in the Getting Started section for now. It's actually not that uncommon to include tutorial content with getting started content. Ex http://guides.rubyonrails.org/getting_started.html. We eventually do want to simplify the getting started content that we have and make it cross-platform, but until we have a cross-platform scaffolding solution for MVC that's difficult to do. I've opened https://github.com/aspnet/Docs/issues/896 to track making the Getting Started content cross-platform.
